src/java.base/share/classes/sun/security/util/DerValue.java

```diff
@@ -302,6 +302,34 @@ public static DerValue wrap(byte tag, DerOutputStream out) {
         return new DerValue(tag, out.buf(), 0, out.size(), false);
     }
 
+    /**
+     * Wraps a byte array at a single DerValue.
+     *
+     * Attention: no cloning is made.
+     *
+     * @param buf the byte array containing the DER-encoded datum
+     * @returns a new DerValue
+     */
+    public static DerValue wrap(byte[] buf)
+            throws IOException {
+        return wrap(buf, 0, buf.length);
+    }
+
+    /**
+     * Wraps a byte array at a single DerValue.
```

Comment:
nit: at=>as

Reply:
There is already a pkg private constructor with exactly same list of arguments. What is the reason for adding a new public static method vs change the pkg private constructor to public?

Reply:
I just want to give it a new name so a user knows the content is only wrapped but not cloned.